CONTRIBUTING.md

```diff
@@ -49,15 +49,16 @@ Guidelines for bug reports:
 ## Feature requests
 
 Feature requests are welcome. But take a moment to find out whether your idea
```

Comment:
Starting a sentence with "But", 😱 

While you're doing these changes, could you change "...welcome. But..." to "...welcome, but..." ?